src/id-api.js

```diff
@@ -16,56 +16,20 @@ const validateIdInfo = (idInfo) => {
   }
 };
 
-const configureJson = ({
-  api_key, id_info, partner_id, partner_params, timestamp,
-}) => JSON.stringify({
+const configurePayload = ({
+  api_key, id_info, partner_id, partner_params,
+}) => ({
   language: 'javascript',
   partner_id,
   partner_params: {
     ...partner_params,
     job_type: parseInt(partner_params.job_type, 10),
   },
   ...id_info,
-  ...new Signature(partner_id, api_key).generate_signature(timestamp),
+  ...new Signature(partner_id, api_key).generate_signature(),
```

Comment:
why was `timestamp` removed?

Reply:
Timestamp was generated in submit_job and then passed all the way down to signature. With this, signature should be the only place concerned with the timestamp generation.

Reply:
looks like what is happening here is that we are no longer permitting end users of the library to pass in a timestamp.

is this the desired design?

Reply:
That is right. Unless partners have a use case to supply timestamp, we just generate the timestamp as part generating the signature.

Reply:
@tamssokari see https://github.com/smileidentity/smile-identity-core-js/pull/50/files#diff-78ed8d9f3cb27f9f99e392f90cf66a417dfbd0c8f9e391c68f5deed27ad14b7dL84 It looks like it was not configurable by the partner.